live-usb-creator/README.md

```diff
@@ -73,17 +94,46 @@ In general there are 7 phases of the build process, all kicked off in sequence w
 
 One of the most time-intensive (and quiet) stages of this process is the `mksquashfs` operation (`Creating 4.0 filesystem on /var/tmp/tmp??????/images/install.img, block size 131072.`). You can get a rough check on progress by comparing `install.img` size to `install.img` size in a previous build, in a separate `vagrant ssh` session (will output some value between 0.0% and 100.0%, where 100.0% is completely done with the creation of the install image, assuming output sizes are in fact identical):
 
-```
+```bash
 sudo sh -c "stat --printf='%s' /var/tmp/tmp*/images/install.img | python -c 'import sys; print \"%0.1f%%\" % (float(int(sys.stdin.read())) / 1045090304.0 * 100.0)'"
 ```
 
 ## Hacking on this project
+
 Here are the useful files you may want to tweak.
 
 VM related stuff:
+
 * `Vagrantfile`: Provisions a VirtualBox VM in which the live system ISO will be built using the Vagrant tool.
 * `bootstrap.sh`: A shell script that runs at the very end of VM bringup. (Currently just mounts the CodeSafe ISO.)
-* `build.sh`: Manually run by the user to kick off the ISO build. Currently includes some patches for Lorax and Kickstart files applied before running the build proper with `livemedia-creator`.
+* `build.sh`: Manually run by the user to kick off the ISO build. Currently includes some patches for Lorax and Kickstart files applied before running the build proper with `livemedia-creator`. Two image types, development and release, can be built.
+  * Release build: `build.sh` or `build.sh release`.
+
+    A release image is to be used by cold storage operators who do not need
+    to have access to an interactive shell. For a release image to boot into
+    a root shell successfully, an SCSI/ATA hard drive needs to be available,
+    with a partition '/dev/sda1' used as persistent storage for the cold
+    wallet. The partition can be formatted in ext3 or vfat for example. The
+    device node `/dev/sda1` is required to give the operators a seamless
+    experience when booting up the system. A few other files, including the
+    HSM world and encrypted secrets, and compiled subzero code (in .jar and
+    signed .sar), are not checked in to the project repository, and need to
+    be added to their appropriate locations before building a release image
+    that will work out of the box.
+
+    * `data_app_subzero/` should contain `subzero-cli.jar`,
```

Comment:
The way this is written, it sounds like you need to put `data_app_subzero` on /dev/sda1, which isn't true.

Reply:
It's good feedback. I will re-write the paragraph in a (hopefully) clearer way.